src/command.ts

```diff
@@ -50,7 +51,15 @@ export async function runCommand<T extends ArgsDef = ArgsDef>(
           );
         }
         const subCommand = await resolveValue(subCommands[subCommandName]);
-        if (subCommand) {
+        // Same behavior as for runMain (see src/main.ts)
+        if (opts.rawArgs.includes("--version")) {
```

Comment:
Maybe, I should move it to another PR.

and should I create a function to avoid duplication with https://github.com/unjs/citty/blob/1cb60fe5b5aa7be64206e8d865b847adafae6549/src/main.ts#L22C7-L22C7 ?